plugins/extsearch/engines/JPopsuki.php

```diff
@@ -13,16 +14,17 @@ class jpopsukiEngine extends commonEngine
 		"TV-Drama"=>"&filter_cat[7]=1",
 		"Fansubs"=>"&filter_cat[8]=1",
 		"Pictures"=>"&filter_cat[9]=1",
-		"Misc"=>"&filter_cat[10]=1" );
+		"Misc"=>"&filter_cat[10]=1"
+		);
 
 	public function action($what,$cat,&$ret,$limit,$useGlobalCats)
 	{
 		$added = 0;
-		$url = 'http://jpopsuki.eu';
+		$url = 'https://jpopsuki.eu';
 		if($useGlobalCats)
-			$categories = array( 'all'=>'0', 
-				'tv'=>"&filter_cat[5]=1&filter_cat[6]=1&filter_cat[7]=1", 
-				'pictures'=>"&filter_cat[9]=1", 
+			$categories = array(
+				'all'=>'',
+				'music'=>''
```

Comment:
Why global category 'tv' is absent? It present in local categories. Why we have same 'all' and 'music' global categories?

Reply:
Yes you are right, fixed.